components/Chips/examples/ChipsFieldDeleteEnabledViewController.swift

```diff
@@ -20,36 +20,42 @@ import MaterialComponentsBeta.MaterialChips_Theming
 import MaterialComponentsBeta.MaterialContainerScheme
 
 class ChipsFieldDeleteEnabledViewController : UIViewController, MDCChipFieldDelegate {
-  var colorScheme = MDCSemanticColorScheme()
-  var shapeScheme = MDCShapeScheme()
-  var typographyScheme = MDCTypographyScheme()
+  var containerScheme = MDCContainerScheme()
   var chipField = MDCChipField()
 
-  var scheme: MDCContainerScheming {
-    let scheme = MDCContainerScheme()
-    scheme.colorScheme = colorScheme
-    scheme.shapeScheme = shapeScheme
-    scheme.typographyScheme = typographyScheme
-    return scheme
-  }
-
   init() {
     super.init(nibName: nil, bundle: nil)
+    commonInit()
   }
 
   @available(*, unavailable)
   required init?(coder aDecoder: NSCoder) {
     super.init(coder: aDecoder)
+    commonInit()
+  }
+
+  func commonInit() {
+    setUpContainerScheme()
+  }
+
+  func setUpContainerScheme() {
+    containerScheme.colorScheme = MDCSemanticColorScheme()
```

Comment:
Please use `MDCColorSchemeDefaultsMaterial201804` to be consistent with the majority of examples. Same for Typography.  Here and throughout this example.